### ISSUE TITLE
Unable to input full-width characters

### ISSUE THREAD

**Issue by Pandaft** [magmax/python-inquirer#432]
**Environment:**

- Windows 10 22H2
- Python 3.11.4
- python-inquirer 3.1.3

**Code:**

```python
import inquirer

answers = inquirer.prompt([
    inquirer.Text('msg',
                  message="message"),
])
```

**Reproduce:**

When I try to type Chinese:

![1](https://github.com/magmax/python-inquirer/assets/80463781/62210e8f-0919-433a-873f-366420e6ac2c)

It turned out:

![2](https://github.com/magmax/python-inquirer/assets/80463781/991adce3-a9c1-4ee4-8445-9c3c5f5a6cae)

This problem also occurs with other full-width characters, such as Korean and Japanese.

**Comment by Cube707** [comment]
This stems from the libary function `readchar` using only the `getch()` function, which only returns single bytes (Its old, from a time where ascii was the only characterset).

There are plans to fix that in an upcoming version, but there's no timeframe on when I will come around to it.

**Comment by Cube707** [comment]
Relevant: https://github.com/magmax/python-readchar/issues/93

**Comment by Cube707** [comment]
closed by #622
